composer.json

```diff
@@ -18,13 +18,13 @@
         "issues": "https://github.com/nstapelbroek/culpa-laravel-5/issues"
     },
     "require": {
-        "illuminate/support": "~5.1",
-        "illuminate/config": "~5.1",
-        "illuminate/events": "~5.1"
+        "illuminate/support": "~6.0",
```

Comment:
Since Laravel 6 is using semver now we can use the `^` sign instead of the `~`. Let's replace that in all illuminate dependencies.